libraries/chain/apply_context.cpp

```diff
@@ -207,6 +207,26 @@ bool apply_context::is_account( const account_name& account )const {
    return nullptr != db.find<account_object,by_name>( account );
 }
 
+void apply_context::get_code_hash(
+   account_name account, uint64_t& code_sequence, fc::sha256& code_hash, uint8_t& vm_type, uint8_t& vm_version) const {
+
+   auto obj = db.find<account_metadata_object,by_name>(account);
+   if(!obj || obj->code_hash == fc::sha256{}) {
```

Comment:
I think this could be simplified to just `if(!obj)` now.

Reply:
The check is so if a new vm_type or vm_version are added in the future, these will still report 0 when there is no contract